### PR TITLE
Allow gff/gff3/vcf in sortBed, single_quote paths and upgrade to bedt…

### PR DESCRIPTION
**Pull request by mvdbeek**
…ools 2.26.0, drop TS dependency, reset  in wrapper version.

### CODE REVIEW

File: tools/bedtools/macros.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <macros>
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="2.24">bedtools</requirement>
+            <requirement type="package" version="2.26.0">bedtools</requirement>
             <yield/>
         </requirements>
     </xml>
-    <token name="@WRAPPER_VERSION@">2.24</token>
+    <token name="@WRAPPER_VERSION@">2.26</token>
```
Comment (nsoranzo) [inline tools/bedtools/macros.xml]: Should we also use 2.26.0 here?

**Comment by nsoranzo** [comment]
Some tests failed, it looks like that upgrading to 2.26.0 creates differences in some test output.

**Comment by mvdbeek** [comment]
Yes, there are some off by one fixes mentioned in the 2.26.0 release notes, though it should only affect vcf/gff test-data. I am running planemo locally with --update_test_data to see what those differences are.

File: tools/bedtools/macros.xml (hunk 1 above)
Comment (mvdbeek) [inline tools/bedtools/macros.xml]: I think so, good catch.

**Comment by bgruening** [comment]
mh, `bedtools_shufflebed` failed :(

**Comment by mvdbeek** [comment]
yeah, it works repeatedly on my local OSX, I'm guessing the new way of shuffling may result in a different outcome based on the platform.

**Comment by mvdbeek** [comment]
Alright, tests pass,  but I have some more commits on my machine in the lab for quoting everything. I'll update this tomorrow.

**Comment by mvdbeek** [comment]
Alright, this is ready to go.

**Comment by bgruening** [comment]
Thanks Marius!

**Comment by mvdbeek** [comment]
Seems no tests were run: https://travis-ci.org/galaxyproject/tools-iuc/builds/168227744, and subsequently no upload :(

**Comment by nsoranzo** [comment]
@mvdbeek That's my fault, should be fixed in https://github.com/galaxyproject/tools-iuc/pull/998 .

**Pull request merged**